test

```diff
@@ -203,7 +202,7 @@ function integration_pass {
 
 function integration_extra {
 	go test -timeout 1m -v ${RACE} -cpu "${TEST_CPUS}" "$@" "${REPO_PATH}/client/integration"
-	go test -timeout 25m -v ${RACE} -cpu "${TEST_CPUS}" "$@" "${REPO_PATH}/clientv3/integration"
+	go test -timeout 25m -v ${RACE} -cpu "${TEST_CPUS}" "$@" "${REPO_PATH}/clientv3/integration/..."
```

Comment:
Curious why `clientv3/integration/...` and `client/integration` are listed here as well as under `integration_pass`.  It looks like duplication at first glance but I must be missing something?

Reply:
Note that release-3.4 has big difference with release-3.5 and main in term of code structure. Both `client` and `clientv3` exist in 3.4.

Reply:
Thanks @ahrtr got that.  Sorry re-reading this I was missing line `189`,  looks like `integration_extra` is what runs if neither `TESTCASE` or `PKG` is set.  No duplication then, disregard this.